crates/bevy_ecs/src/schedule_v3/executor/single_threaded.rs

```diff
@@ -0,0 +1,156 @@
+#[cfg(feature = "trace")]
+use bevy_utils::tracing::{info_span, Instrument};
+use fixedbitset::FixedBitSet;
+
+use crate::{
+    schedule_v3::{is_apply_system_buffers, SystemExecutor, SystemSchedule},
+    world::World,
+};
+
+/// Runs the schedule using a single thread.
```

Comment:
```suggestion
/// Runs the schedule using a single thread.
///
/// This may be useful when parallelizing many schedules at once, when working on a single-threaded execution environment, or when you have alternative strategies for parallelism.
```